protocols/protocol/cloudstate/entity.proto

```diff
@@ -164,27 +263,69 @@ message Entity {
     string persistence_id = 3;
 }
 
+//UserFunctionError contains information about a problem pretaining to erronous behavior of a UserFunction.
+//
+//---
+//
+//`Specification`:
+//
+//  - A `UserFunctionError` is considered to be valid iff:
+//    - `message` MUST have length >= 1
 message UserFunctionError {
+    // A human-readable error message.
     string message = 1;
 }
 
+//ProxyInfo contains information about the CloudState PRoxy which is responsible for proxying a UserFunction.
+//
+//---
+//
+//`Specification`:
+//
+//  - A `ProxyInfo` is considered to be *valid* iff:
+//    - `protocol_major_version` >= 0
+//    - `protocol_minor_version` >= 0
+//    - `proxy_name` TODO
+//    - `proxy_version` TODO
+//    - `supported_entity_types` is one of the following values:
+//      - TODO
+//      - TODO
+//      - TODO
 message ProxyInfo {
+    // TODO
     int32 protocol_major_version = 1;
+    // TODO
     int32 protocol_minor_version = 2;
+    // TODO
     string proxy_name = 3;
+    // TODO
     string proxy_version = 4;
+    // TODO
     repeated string supported_entity_types = 5;
 }
 
-// Entity discovery service.
+//The *EntityDiscovery* Service is the entrypoint for CloudState Proxies
```

Comment:
punctuation at the end of the sentence.
EntityDiscovery is bold here, whereas messages above are not. Is that intentional?

(I like the consistent naming of the message names/services that follow the spec-comment; also punctuation)

Reply:
I had a different idea for Services vs Messages, I'm not going to die on that hill though :)